### PR TITLE
Stops flamethrower crazed gorillas and gives them identifying numbers

### LINKED ISSUE #30104: Gorillas can use guns and don't have identifier numbers
WHAT THE FUCK. LETS JUST MAKE SUPER HULKS THAT DONT BLEED AND CANT LOSE LIMBS BUT CAN ALSO USE GUNS AND STUN BATONS

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #30104

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm [hunk 1]
```diff
@@ -75,3 +76,6 @@
 		playsound(src, "sound/creatures/gorilla.ogg", 200)
 	..()
 
+/mob/living/simple_animal/hostile/gorilla/can_use_guns(var/obj/item/G)
```
Comment (lzimann) [inline code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm]: no `var/` at the argument

**Comment by Kromgar** [comment]
Speedmerge pls

**Review by Cyberboss — APPROVED** [review]

**Comment by ShizCalev** [comment]
Ignore the locking and all that, was testing something w/ CB

**Comment by ShadowDeath6** [comment]
rip the coconut gun dream

**Comment by FantasticFwoosh** [comment]
You can make the coconut gun unrestricted @ShadowDeath6 instead

**Pull request merged**